backend/Tim.Backend/Program.cs

```diff
@@ -42,11 +43,6 @@ public static IWebHost BuildWebHost(string[] args)
                 .ConfigureAppConfiguration((config) =>
                 {
                     config.Build();
-
-                    if (Env.CurrentEnvironment == RuntimeEnvironment.PPE)
-                    {
-                        config.AddUserSecrets<AzureResourcesSectionSecrets>();
```

Comment:
this is nice fore debugging since it actually keeps your secrets from being checked into the branch if we do store secrets in environmental variables

Reply:
this is what reads values from secrets.json file and puts them into the object you give it.

Reply:
the `secrets.json` are read by default, no need to add anything extra to use them

I'm currently using them without this code.